apidoc/Titanium/UI/WebView.yml

```diff
@@ -432,6 +432,48 @@ methods:
     returns:
         type: BackForwardList
 
+  - name: createPDF
+    summary: |
+        Create a PDF document representation from the web page currently displayed in the WebView.
+    description: |
+        If the data is written to a file the resulting file is a valid PDF document.
+    platforms: [iphone, ipad]
+    since: "9.2.0"
+    osver: {ios: {min: "14.0"}}
+    parameters:
+      - name: callback
+        summary: Function to call upon pdf creation.
+        type: Callback<DataCreationResult>
+
+  - name: createWebArchive
+    summary: Create WebKit web archive data representing the current web content of the WebView.
+    description: |
+        WebKit web archive data represents a snapshot of web content. It can be loaded into a WebView directly, 
```

Comment:
It's unclear to me from the apple docs or looking at our code exactly how this would be used to load into a web view.

Presumably we can save this to a file, great - but how do we then load the file or blob and set it in a web view later? Can we simply provide the `Ti.Filesystem.File` or `Ti.Blob` we received as the `Ti.UI.WebView.data` property and it magically works? If so, perhaps we can provide an example in the docs here and add a unit test which does so?